modal/functions.py

```diff
@@ -1394,6 +1391,32 @@ async def coro():
                 obj.enter()
                 return fun(*args, **kwargs)
 
+    @synchronizer.no_input_translation
+    @live_method
+    async def _experimental_spawn(self, *args: P.args, **kwargs: P.kwargs) -> "_FunctionCall[ReturnType]":
+        """[Experimental] Calls the function with the given arguments, without waiting for the results.
+
+        This experimental version of the spawn method allows up to 1 million inputs to be spawned.
+
+        Returns a `modal.functions.FunctionCall` object, that can later be polled or
+        waited for using `.get(timeout=...)`.
+        Conceptually similar to `multiprocessing.pool.apply_async`, or a Future/Promise in other contexts.
+
+        *Note:* `.spawn()` on a modal generator function does call and execute the generator, but does not currently
```

Comment:
Slightly unrelated but this note from the docstring of `spawn` is no longer true - we always return a FunctionCall object, even for generators (that can fetch results using `.get_gen()`), so we should remove this note (from spawn too)

Reply:
ah good to know. I'll remove from both.